src/test/bridges/example/ExampleUnit.t.sol

```diff
@@ -9,14 +9,16 @@ import {AztecTypes} from "../../../aztec/libraries/AztecTypes.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {ExampleBridgeContract} from "../../../bridges/example/ExampleBridge.sol";
 import {ErrorLib} from "../../../bridges/base/ErrorLib.sol";
+import {ISubsidy, Subsidy} from "../../../aztec/Subsidy.sol";
 
 // @notice The purpose of this test is to directly test convert functionality of the bridge.
 contract ExampleUnitTest is Test {
-    IERC20 public constant DAI = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
-
-    AztecTypes.AztecAsset internal emptyAsset;
+    address private constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address private constant BENEFICIARY = address(11);
```

Comment:
Using a never before used account means that you will be paying extra gas for creating the account in account trie. I think it will be more expected that the beneficiary address have been used before and is reused. Consider making a transfer to the `BENEFICIARY` in setup or finding another address.

Reply:
Addressed in cf8793d261feb6282b2a89f3dbc747281e16f21a + compared gas costs of empty and non-empty addresses in the PR comment.